starter-code/french-onion-soup-starter-code/index.html

```diff
@@ -1,43 +1,60 @@
   <title>French Onion Soup Recipe</title>
   <!-- Google fonts gives us access to the Open Sans Font -->
   <!-- Google fonts is also awesome! Make sure to check it out -->
   <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,600" rel="stylesheet">
 </head>
 <body>
-  French Onion Soup
+<figure>
+      <img class ="img" src="images/soup.jpg" alt="soup image">
+</figure>
+      <h1 class = "title">French Onion Soup</h1>
 
-  We have been trying French onion soup in restaurants for years and my family
-and friends agree none can compare to my recipe for taste and simplicity of
-preparation.
 
-Ingredients
-1/2 cup unsalted butter
-2 tablespoons olive oil
-4 cups sliced onions
-4 (10.5 ounce) cans beef broth
-2 tablespoons dry sherry (optional)
-1 teaspoon dried thyme
-salt and pepper to taste
-4 slices French bread
-4 slices provolone cheese
-2 slices Swiss cheese, diced
-1/4 cup grated Parmesan cheese
+  <p class = "p1">We have been trying French onion soup in restaurants for years and my family
+and friends agree none can compare to my recipe for taste and simplicity of
+preparation.</p>
+<figure>
+      <img class ="img2" src="images/recipe-info.png" alt="recipe image">
+</figure>
+<hr width="65%" size ="2px" color ="grey"/>
+<h2 class="title2">Ingredients</h2>
+<ul class="list">
+<li>1/2 cup unsalted butterc</li>
+<li>2 tablespoons olive oil</li>
+<li>4 cups sliced onions</li>
+<li>4 (10.5 ounce) cans beef broth</li>
+<li>2 tablespoons dry sherry (optional)</li>
+<li>1 teaspoon dried thyme</li>
+<li>salt and pepper to taste</li>
+<li>4 slices French bread</li>
+<li>4 slices provolone cheese</li>
+<li>2 slices Swiss cheese, diced</li>
+<li>1/4 cup grated Parmesan cheese</li>
+</ul>
+<figure>
+      <img class ="img3" src="images/cooking-info.png" alt="cooking image">
+</figure>
+<hr width="65%" size ="2px" color ="grey"/>
+<h2 class="title2">Directions</h2>
 
-Directions
-Melt butter with olive oil in an 8 quart stock pot on medium heat. Add onions and
-continually stir until tender and translucent. Do not brown the onions.
+<ol class="list">
+<li>Melt butter with olive oil in an 8 quart stock pot on medium heat. Add onions and
+continually stir until tender and translucent. Do not brown the onions.</li>
 
-Add beef broth, sherry and thyme. Season with salt and pepper, and simmer for 30
+<li>Add beef broth, sherry and thyme. Season with salt and pepper, and simmer for 30
 minutes.
 
-Heat the oven broiler.
+<li>Heat the oven broiler.</li>
 
-Ladle soup into oven safe serving bowls and place one slice of bread on top of each
+<li>Ladle soup into oven safe serving bowls and place one slice of bread on top of each
 (bread may be broken into pieces if you prefer). Layer each slice of bread with a slice of
 provolone, 1/2 slice diced Swiss and 1 tablespoon Parmesan cheese. Place bowls on
 cookie sheet and broil in the preheated oven until cheese bubbles and browns slightly.
+</li>
+</ol>
 </body>
 </html>
```

Comment:
The overall structure is good, you should try to center and align some parts like the main image or the text